Buildings/Controls/OBC/ASHRAE/G36_PR1/Generic/SetPoints/ZoneStatus.mo

```diff
@@ -1,380 +1,420 @@
 within Buildings.Controls.OBC.ASHRAE.G36_PR1.Generic.SetPoints;
 block ZoneStatus "Block that outputs zone temperature status"
 
+  parameter Real THeaSetOcc(
+    final unit="K",
+    final displayUnit="degC",
+    final quantity="ThermodynamicTemperature")=293.15
+    "Occupied heating setpoint";
+  parameter Real THeaSetUno(
+    final unit="K",
+    final displayUnit="degC",
+    final quantity="ThermodynamicTemperature")=285.15
+    "Unoccupied heating setpoint";
+  parameter Real TCooSetOcc(
+    final unit="K",
+    final displayUnit="degC",
+    final quantity="ThermodynamicTemperature")=297.15
+    "Occupied cooling setpoint";
+  parameter Real TCooSetUno(
+    final unit="K",
+    final displayUnit="degC",
+    final quantity="ThermodynamicTemperature")=303.15
+    "Unoccupied cooling setpoint";
   parameter Real bouLim(
     final unit="K",
     final displayUnit="K",
     final quantity="TemperatureDifference",
-    final min=0.5) = 1.1
-    "Value limit to indicate the end of setback or setup mode";
+    final min=0.5) = 1
+    "Threshold of temperature difference for indicating the end of setback or setup mode";
   parameter Boolean have_winSen=false
```

Comment:
should we change this parameter to have_winSen = have_winSen so that it can be propagated up?

Reply:
`have_winSen` is already a parameter here. We don't need to propagate it here. However, we would need to propagate it in blocks using the `ZoneStatus` to compose. Like the block `ModeAndSetPoints` in single zone AHU package, it has been propagated.